### DISCUSSION TITLE
Is there a way to alter the pgcat application_name so it displays in the pg_stat_activity instead of just "pgcat"?

### DISCUSSION THREAD

**Discussion by toutas** [postgresml/pgcat#888] [Q&A]
I have been attempting to set application_name in the toml config file, but it seems like this is not possible?

**Comment by drdrsh** [comment]
PgCat will set `application_name` to whichever value is set on the client. So you just need to set `application_name` on your client connection and PgCat will propagate that to the server.

**Reply by toutas** [reply]
makes sense, thanks!
